### PR TITLE
Feature/arrays

### PR DESCRIPTION
**Pull request by luislourencos**

### CODE REVIEW

File: staff/sergio-luis/arrays/array-functions/every.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function every(array, expression) {
+
+    for (var i = 0; i < array.length; i++) {
+      if (!expression(array[i])) {
```
Comment (Dashkent) [inline staff/sergio-luis/arrays/array-functions/every.js:4]: it should also take the optional parameters

File: staff/sergio-luis/arrays/array-functions/filter.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+function filter(array, expresion) {
+    var result = [];
+    for (var i = 0; i < array.length; i++) {
+        if(expresion(array[i])){
```
Comment (Dashkent) [inline staff/sergio-luis/arrays/array-functions/filter.js:4]: it should also take the optional parameters

File: staff/sergio-luis/arrays/array-functions/every.js (hunk 1 above)
Comment (Aitor1111) [inline staff/sergio-luis/arrays/array-functions/every.js:4]: Great work! You can also include index and array to the expression: ```(!expression(array[i], i, array))```as the every method lets to include.

File: staff/sergio-luis/arrays/array-functions/filter.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+            result[result.length]= array[i]
+        };
+    }
```
Comment (Aitor1111) [inline staff/sergio-luis/arrays/array-functions/filter.js:7]: Just a suggestion: in for loop and in the if statement you can remove curly braces and make code less large. Looks like:
```for (var i = 0; i < array.length; i++) if(expresion(array[i])) result[result.length]= array[i]```

Comment (Aitor1111) [reply staff/sergio-luis/arrays/array-functions/filter.js:7]: Manu does this in forEach example, take a look

Comment (luislourencos) [reply staff/sergio-luis/arrays/array-functions/filter.js]: Ok thanks! i will make the change! <3

File: staff/sergio-luis/arrays/array-functions/every.js (hunk 1 above)
Comment (luislourencos) [reply staff/sergio-luis/arrays/array-functions/every.js]: ok! done!

Comment (luislourencos) [reply staff/sergio-luis/arrays/array-functions/every.js]: Ok i make the change! but i have some dudes of to pass the optional parameters

File: staff/sergio-luis/arrays/array-functions/filter.js (hunk 2 above)
Comment (luislourencos) [reply staff/sergio-luis/arrays/array-functions/filter.js]: Ok! thanks tomorrow i will ask to Alex!

**Pull request merged**